index.js

```diff
@@ -87,8 +87,8 @@ function pinoMultiStream (opts, stream) {
 Object.assign(pinoMultiStream, pino)
 pinoMultiStream.multistream = multistream
 pinoMultiStream.prettyStream = (args = {}) => {
-  const { opts = {}, prettifier, dest = process.stdout } = args
-  return getPrettyStream(opts, prettifier, dest)
+  const { prettyPrint = {}, prettifier, dest = process.stdout } = args
```

Comment:
This is a breaking change.

Reply:
Can you make it a non-breaking change? You'll need to also support `opts`.

Reply:
> Can you make it a non-breaking change? You'll need to also support `opts`.
Right. I've updated the PR.